examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -4398,10 +4399,11 @@ endpoint 1 {
     ram      attribute minSetpointDeadBand default = 0x19;
     ram      attribute controlSequenceOfOperation default = 0x04;
     ram      attribute systemMode default = 0x01;
+    ram      attribute thermostatRunningMode;
     ram      attribute startOfWeek;
     ram      attribute numberOfWeeklyTransitions default = 7;
     ram      attribute numberOfDailyTransitions default = 4;
-    ram      attribute featureMap default = 0x000b;
+    ram      attribute featureMap default = 0x0023;
```

Comment:
So this used to claim HEAT, COOL, SCH.

The new value claims HEAT, COOL, AUTO.

Why was SCH removed?

Reply:
since we do not support scheduling. i can add it back.